lib/internal/bootstrap_node.js

```diff
@@ -48,6 +48,11 @@
     if (process.argv[1] !== '--debug-agent')
       _process.setupChannel();
 
+    // TODO(evanlucas) Remove this when v8_inspector is no longer experimental.
+    if (process._useInspector) {
+      process.emitExperimentalWarning('v8_inspector');
+    }
```

Comment:
This should be after `setupRawDebug()` at minimum.

However, I think this probably needs to happen at the end of the first tick?

i.e. do we want user handlers to catch this?